ssl/ssl_lib.c

```diff
@@ -2506,6 +2506,24 @@ STACK_OF(SSL_CIPHER) *SSL_CTX_get_ciphers(const SSL_CTX *ctx)
     return NULL;
 }
 
+/* Distinguish between ciphers controlled by set_ciphersuite() and
+ * set_cipher_list() when counting.
+ */
+static int cipher_list_num(STACK_OF(SSL_CIPHER) *sk) {
```

Comment:
Would recommend that the function name indicate that it only counts TLSv1.2 (or earlier ciphers) to make the use of the function easier to under stand. e.g. `cipher_tlist_ls12_num()`.